tests/unit/command_test.ts

```diff
@@ -59,7 +59,13 @@ Deno.test(
     const command = new Deno.Command(Deno.execPath(), {
       args: [
         "eval",
-        "if (new TextDecoder().decode(await Deno.readAll(Deno.stdin)) !== 'hello') throw new Error('Expected \\'hello\\'')",
+        `
+        const buffer = new Uint8Array(5);
+        await Deno.stdin.read(buffer);
```

Comment:
While this passes today, I don't think this is guaranteed to always work and return 5. We are probably going to flake in the future.

Reply:
Unfortunately that's not true.

The problem is that the read isn't guaranteed to read the entire buffer's worth of data. It could read 1, 2, 3, 4 or 5 chars. It _might_ work for now but it's possible this is a test flake waiting to happen.

The only guaranteed read size that will always fill a buffer is 1 -- you could just read 5 times in a row and that would guarantee that you get 5 chars.

Reply:
While I agree with @mmastrac here, it's worth pointing out that we have _many_ tests that do the exact same thing, and they haven't really been flaky. Would it be safe to assume that for sufficiently small buffers (<100 bytes) a single read would fill the entire buffer in a single syscall?

Reply:
I think it's safe to assume a single read fills the buffer for small buffers. Yeah, I'd agree with you guys that this wouldn't be a good idea if the buffer were larger. Either way, I'm happy to use `readAll()` from `std/io` if you guys feel strongly about this.

Reply:
Let's try to use readAll in the future. For now let's get this landed, but ideally we should not use read anywhere in testing.